src/main/java/com/cburch/logisim/std/wiring/Pin.java

```diff
@@ -253,6 +253,148 @@ public void keyReleased(KeyEvent e) {}
     public void keyTyped(KeyEvent e) {}
   }
 
+  @SuppressWarnings("serial")
```

Comment:
Please add serial instead.

Reply:
I had assumed copying `EditDecimal` would be fine
Should it be updated as well?

Reply:
Perhaps (as the less suppresses the better), but I'd suggest not doing that in scope of this PR.

Reply:
Removed now